app/Example.java

```diff
@@ -0,0 +1,50 @@
+import android.view.View;
+
+import io.ably.lib.realtime.AblyRealtime;
+import io.ably.lib.realtime.Channel;
+import io.ably.lib.types.AblyException;
+import io.ably.lib.types.Message;
+import io.ably.lib.types.Param;
+
+public class Example {
+
+	private static final String API_KEY = "INSERT-YOUR-API-KEY-HERE";
```

Comment:
Can you raise a nice error message for users when this key is not set like we do in the other tutorials?  This will stop people wondering why it does not "just work", see https://github.com/ably/tutorials/blob/publish-subscribe-ruby/example.rb#L4 for an example.